wp-includes/rest-api/endpoints/class-wp-rest-customize-partials-controller.php

```diff
@@ -0,0 +1,327 @@
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since ?.?.?
+ */
+
+/**
+ * Core class to access partials via the REST API.
+ *
+ * @since ?.?.?
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Customize_Partials_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 4.7.0
+	 * @access public
+	 */
+	public function __construct() {
+		$this->namespace = 'customize/v1';
+		$this->rest_base = 'partials';
+	}
+
+	/**
+	 * Ensure customize manager.
+	 *
+	 * @return WP_Customize_Manager Manager.
+	 * @global WP_Customize_Manager $wp_customize
+	 */
+	public function ensure_customize_manager() {
+		global $wp_customize;
+		if ( empty( $wp_customize ) ) {
+			$wp_customize = new WP_Customize_Manager(); // WPCS: global override ok.
+		}
+		if ( ! did_action( 'customize_register' ) ) {
+
+			/** This action is documented in wp-includes/class-wp-customize-manager.php */
+			do_action( 'customize_register', $wp_customize );
+		}
+		return $wp_customize;
+	}
+
+	/**
+	 * Registers the routes for the objects of the partialler.
+	 *
+	 * @since ?.?.?
+	 * @access public
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes() {
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'                => $this->get_collection_params(),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/(?P<partial>[\w-]+)', array(
+			'args' => array(
+				'partial' => array(
+					'description' => __( 'An alphanumeric identifier for the partial.' ),
+					'type'        => 'string',
+				),
+			),
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_item' ),
+				'permission_callback' => array( $this, 'get_item_permissions_check' ),
+				'args'                => array(
+					'context' => $this->get_context_param( array(
+						'default' => 'view',
+					) ),
+				),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	/**
+	 * Retrieves the partial's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.?.?
+	 * @access public
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/schema#',
+			'title'      => 'partial',
+			'type'       => 'object',
+			'properties' => array(
+				'fallback_refresh' => array(
+					'description' => __( 'Whether to refresh the entire preview in case a partial cannot be refreshed.' ),
+					'type'        => 'boolean',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'id'              => array(
```

Comment:
Currently the properties are in alphabetical order in every endpoint in this plugin. Checked that in `posts` and `revisions` endpoints the properties are in alphabetical order as well, in `users` and `terms` `id` is the first. Do you think it would be better to have `id` as the first in every endpoint?